extensions/amp-story/1.0/_locales/de.js

```diff
@@ -28,19 +28,19 @@ export default {
     string: 'Ablehnen',
   },
   [LocalizedStringId.AMP_STORY_DOMAIN_DIALOG_HEADING_LABEL]: {
-    string: 'Sehen Sie auf der ursprünglichen Domäne',
+    string: 'Auf der originalen Webseite anschauen',
   },
   [LocalizedStringId.AMP_STORY_DOMAIN_DIALOG_HEADING_LINK]: {
-    string: 'Mehr über die AMP Ergebnisse',
+    string: 'Mehr Informationen zum Thema AMP Ergebnisse',
```

Comment:
"zum Thema" sounds slightly colloquial. How about "Mehr Informationen zu AMP Ergebnissen"?

Reply:
It does not really sound "colloquial" to me....but maybe I am biased here :-)
Your suggestion also sounds good to me, so no prob to use your recommendation.